azurerm/internal/services/mssql/mssql_database_resource.go

```diff
@@ -444,6 +452,30 @@ func resourceMsSqlDatabaseCreateUpdate(d *schema.ResourceData, meta interface{})
 
 	d.SetId(*read.ID)
 
+	// For datawarehouse SKUs, set the geo-backup policy
+	if d.HasChange("geo_backup_policy") && strings.HasPrefix(skuName.(string), "DW") {
+		v := d.Get("geo_backup_policy").(bool)
```

Comment:
should we also only call this API on creation when its set to true (or has a change on update?)

Reply:
The default for Geo backup policy is true, so make the logic so that it will set the property:

- on changes
- on new resource creation where the geo backup enabled property is false.